src/components/Toolbar/VisualizationTypeSelector/VisualizationTypeSelector.js

```diff
@@ -39,12 +45,14 @@ export const VisualizationTypeSelector = ({ visualizationType }) => {
         <Card dataTest={'visualization-type-selector-card'}>
             <div className={classes.listContainer}>
                 <div className={classes.listSection}>
-                    {getVisTypes().map((type) =>
-                        visTypeMap[type].disabled ? (
+                    {[VIS_TYPE_LINE_LIST, VIS_TYPE_PIVOT_TABLE].map((type) =>
+                        type === VIS_TYPE_PIVOT_TABLE ? (
                             <Tooltip
                                 key={`${type}-tooltip`}
                                 placement="bottom"
-                                content={visTypeMap[type].disabledText}
+                                content={i18n.t(
+                                    'Pivot tables are not supported by this app yet'
```

Comment:
as per the comment above, consider `{{visTypeName}} visualizations are not supported by this app yet` for future-proofing

Reply:
Fixed by using a generic text that does not use the vis type.